etc/cmake/options.cmake

```diff
@@ -42,7 +42,9 @@ endif()
 
 option(OTBR_BORDER_ROUTING "Enable Border Routing Manager" OFF)
 
-option(OTBR_BORDER_ROUTING_NAT64 "Enable NAT64 support in Border Routing Manager" OFF)
+option(OTBR_BORDER_ROUTING_NAT64 "Enable NAT64 border routing support [deprecating]" OFF)
+
+option(OTBR_NAT64_BORDER_ROUTING "Enable NAT64 border routing support" OFF)
```

Comment:
I don't think these two options are necessary. We can just use the corresponding OpenThread options.